doc/API.md

```diff
@@ -76,23 +76,25 @@ Each rule in axe-core has a number of tags. These provide metadata about the rul
-| `section508.*.*` | Requirement in old Section 508                       |
-| `TTv5`           | Trusted Tester v5 rules                              |
-| `TT*.*`          | Test ID in Trusted Tester                            |
-| `experimental`   | Cutting-edge rules, disabled by default              |
-| `cat.*`          | Category mappings used by Deque (see below)          |
+| Tag Name         | Accessibility Standard / Purpose                                                                                              |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| `wcag2a`         | WCAG 2.0 Level A                                                                                                              |
+| `wcag2aa`        | WCAG 2.0 Level AA                                                                                                             |
+| `wcag2aaa`       | WCAG 2.0 Level AAA                                                                                                            |
+| `wcag21a`        | WCAG 2.1 Level A                                                                                                              |
+| `wcag21aa`       | WCAG 2.1 Level AA                                                                                                             |
+| `wcag22aa`       | WCAG 2.2 Level AA                                                                                                             |
+| `best-practice`  | Common accessibility best practices                                                                                           |
+| `wcag***`        | WCAG success criterion e.g. wcag111 maps to SC 1.1.1                                                                          |
+| `ACT`            | W3C approved Accessibility Conformance Testing rules                                                                          |
+| `section508`     | Old Section 508 rules                                                                                                         |
+| `section508.*.*` | Requirement in old Section 508                                                                                                |
+| `TTv5`           | Trusted Tester v5 rules                                                                                                       |
+| `TT*.*`          | Test ID in Trusted Tester                                                                                                     |
+| `EN.301.549`     | Rule required under [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf) |
+| `EN-9.*`         | Section in EN 301 549 listing the requirement                                                                                 |
```

Comment:
Using just `EN` without any other prefix makes me nervous about creating future backcompat issues; I think future revisions of EN 301 549 are likely to maintain consistent section numbering, but if there's ever a future different EN standard to, say, cover WCAG 3, it's reasonably likely it'd use conflicting numbering. That leads me to prefer using a fuller prefix here, like `EN-301-549-9.1.2.3` or `EN-301-549.9.1.2.3`.

Reply:
We have that problem with other tags too. `wcag111` won't be relevant when WCAG 3 comes out. `TT1.a` will probably be obsolete when Trusted Tester v6 comes out. Don't quite know what we're going to do at that point but it seems likely we need to come up with something different by that time.

I can well imagine we deprecate the criteria/section specific tags all together and put that information somewhere else. That's what we did with the ACT Rules, we added an `actId` prop to the rule.

Reply:
As discussed, the plan is to eventually not include SC numbers / section numbers into tags but make them available in some other format in the output. For now we're going with what we've been doing up until this point.